### DISCUSSION TITLE
機器人藝術家創作開展

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#136] [General]
Ai-Da是第一位**機器人藝術家**，他是用人工智能（AI）運算法設計而成，並且已創作出一幅自己的自畫像。
看到這篇貼文的時候相當訝異，機器人竟然能創作。
但回到根本，機器人的創作還是來自程式演算法，以設定好的模式製作作品。以Ai-Da的創作為例，她的自畫像雖具所有藝術創作的形式與結構，卻是一幅沒有「自我」的自畫像。
[http://www.cam.org.tw/2021-news62/](url)

**Comment by enoch-lightning** [comment]
我點開你附的網址後，跑出下面這樣的畫面。目前我也不知如何解決
<img width="960" alt="看不到機器人畫家網站" src="https://user-images.githubusercontent.com/86505454/139045123-97c912a0-10fc-4d9b-9af4-85796851aa1f.png">
。

**Comment by 20010906** [comment]
我也出現了404 not found的畫面

**Comment by zixunn** [comment]
我覺得藝術源自於人類的情感，雖然成功讓艾達作畫對人工智慧的研究上是一大突破，然而其作品我認為仍缺少了靈魂與情緒表達。

**Comment by syt-110** [comment]
我覺得繪畫主要還是在情感和處境表達，雖然能創作但離藝術還是有差別

**Comment by HSU-YU-JEN** [comment]
會不會被訓練之後，能夠讓機器人擁有小部分自我意識與情感呢

**Comment by 109302003** [comment]
最近很多的藝術創作都是以機器人為出發點，第一次知道有機器人本身的創作，覺得其中的關係很巧妙，感謝分享～

**Comment by Joan0518** [comment]
這個藝術創作說到底還是當初設計賦予它想像與視覺傳達能力的工程師和設計師，而不是機器人

**Comment by yt7l0** [comment]
比起藝術，他應該更接近數據演算之後的成果吧

**Comment by chupochun** [comment]
這些藝術也不過是，人類創造的人工智慧創造出來的
